terraform/sftp.tf

```diff
@@ -0,0 +1,112 @@
+resource "aws_transfer_server" "sftp" {
+  domain                 = "S3"
+  protocols              = ["SFTP"]
+  endpoint_type          = "PUBLIC"
+  identity_provider_type = "SERVICE_MANAGED"
+  security_policy_name   = "TransferSecurityPolicy-2022-03"
+
+  tags = {
+    Name = "sftp"
+  }
+}
+
+resource "aws_iam_role" "sftp_user" {
+  name               = "transfer-user-iam-role"
+  assume_role_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+        "Effect": "Allow",
+        "Principal": {
+            "Service": "transfer.amazonaws.com"
+        },
+        "Action": "sts:AssumeRole"
+        }
+    ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "sftp_user" {
+  name = "transfer-user-iam-policy"
+  role = aws_iam_role.sftp_user.id
+
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowListingOfUserFolder",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Effect": "Allow",
+            "Resource": [ "${aws_s3_bucket.sftp_storage.arn}" ]
+        },
+        {
+            "Sid": "HomeDirObjectAccess",
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObjectVersion",
+                "s3:DeleteObject",
+                "s3:GetObjectVersion"
+            ],
+            "Resource": ["${aws_s3_bucket.sftp_storage.arn}/*"] 
+        }
+    ]
+}
+POLICY
+}
+
+resource "aws_s3_bucket" "sftp_storage" {
+  bucket = "pcc-integration-data-files-${var.target_env}"
```

Comment:
I just wanted to double check that we're planning to use a different bucket name than the current one of: "bc-pcc-data-files-<env>" ?